platform/feature-api/communication/postoffice/src/types.ts

```diff
@@ -1,52 +1,52 @@
 /**
- * This module defines the types used for specifying endpoints.
+ * This module defines the types used for specifying backend endpoints.
  *
- * An _endpoint_ is an API that can be implemented by a server and called by a
- * client. Endpoints are structured as objects with methods; methods may return
+ * A _backend endpoint_ is an API that can be implemented by a server and called by a
+ * client. Backend endpoints are structured as objects with methods; methods may return
  * either values or a nested endpoint.
  *
  * An _endpoint specification_ is a mere type that is never used directly. See
- * [[EndpointSpec]] for details.
+ * [[BackendSpec]] for details.
  *
  * @packageDocumentation
  */
 
 /**
- * Interface denoting a value endpoint of type `T`. This interface is purely
+ * Interface denoting a value backend endpoint of type `T`. This interface is purely
  * virtual and no instances are generated.
  */
-export interface ValueEndpointSpec<T> {
+export interface ValueBackendSpec<T> {
```

Comment:
I don't think I've seen this before... I don't like it a bit. Defining a type with a string, which is not even part of an enum... Hum. Not part of the PR, but still.